src/sql/workbench/electron-browser/modelComponents/formContainer.component.ts

```diff
@@ -175,6 +175,10 @@ export default class FormContainer extends ContainerBase<FormItemLayout> impleme
 		return itemConfig ? itemConfig.title : '';
 	}
 
+	private hasItemTitle(item: FormItem): boolean {
+		return this.getItemTitle(item) !== '';
```

Comment:
check for is null or undefined too?

Reply:
getItemTitle() will either return the title or an empty string, so it'll take care of null or undefined values and they don't need to be checked for here.